mne/preprocessing/maxwell.py

```diff
@@ -505,3 +539,38 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _overlap_projector(data_int, data_res, delta=0.02):
+    """Calculate projector for removal of subspace intersection in tSSS"""
+    # delta necessary to deal with noise when finding identical signal
+    # directions in the subspace. See the end of the Results section in [2]_
+
+    # Note that the procedure here is an updated version of [2]_ using
+    # residuals instead of internal/external spaces directly. This provides
+    # data with higher rank to work with.
+
+    # Normalize data
+    normed_data_int = data_int / norm(data_int)
+    normed_data_res = data_res / norm(data_res)
+
+    # Compute orth to get temporal bases. Matrices must have shape
+    # (n_samps x effective_rank) when passed into svd computation
+    orth_int = orth(normed_data_int.T)
+    orth_res = orth(normed_data_res.T)
+
+    Q_int, _ = qr(orth_int, mode='economic')
+    Q_res, _ = qr(orth_res, mode='economic')
+
+    C_mat = np.dot(Q_int.T, Q_res)
+
+    # Compute angles between subspace and which bases to keep
+    _, S_intersect, Vh_intersect = svd(C_mat)
```

Comment:
pretty sure you need this to be `svd(..., full_matrices=False), that wil give you the dimensionality you expect later